spring-rabbit/src/test/java/org/springframework/amqp/rabbit/support/micrometer/ObservationTests.java

```diff
@@ -110,7 +111,9 @@ void endToEnd(@Autowired Listener listener, @Autowired RabbitTemplate template,
 
 			@Override
 			public KeyValues getLowCardinalityKeyValues(RabbitMessageSenderContext context) {
-				return super.getLowCardinalityKeyValues(context).and("foo", "bar");
+				return super.getLowCardinalityKeyValues(context).and("foo", "bar")
+					.and("messaging.destination.name", context.getExchange())
+					.and("messaging.rabbitmq.destination.routing_key", context.getRoutingKey());
```

Comment:
I guess this will go to `TemplateLowCardinalityTags` in the next contribution.

Reply:
I think now it's more easier to determine exchange or routing key for doing traces/metrics without splitting `/` in `RabbitMessageSenderContext`.